### DISCUSSION TITLE
Translation to french

### DISCUSSION THREAD

**Discussion by Realpims** [aclap-dev/video-downloadhelper#1421] [Language & Translation]
Hi,
Here is my contribution to translate user interface in French
[fr.txt](https://github.com/user-attachments/files/17587232/fr.txt)

**Comment by Jojaba** [comment]
Hi!
I also translated the missing items in version 9.1.0.44 but additionaly improved some items translation ;)
Best regards, Joël
[fr.txt](https://github.com/user-attachments/files/17612068/fr.txt)

**Comment by LexFH** [comment]
Hello, another brick in the wall : 
[fr.txt](https://github.com/user-attachments/files/17684425/fr.txt)

**Comment by S-e-b-G** [comment]
And another one :)
[fr.txt](https://github.com/user-attachments/files/17688898/fr.txt)

**Comment by Jenjango** [comment]
Hi! :) And here are some small modifications

[fr.txt](https://github.com/user-attachments/files/17924455/fr.txt)

**Comment by magicvince** [comment]
For french translator : Quelques rapprochements des différentes versions suggérant des termes communs et une logique de rendre les traductions plus compactes avec un vocabulaire plus courant

Report = Rapporter (et ses dérivés rapport, rapporté)
Record = enregistrer
Privacy = confidentialité
Entries = données
…
[fr.txt](https://github.com/user-attachments/files/17969391/fr.txt)

**Reply by Jojaba** [reply]
Hello magicvince
FR
¤¤
Comme nous sommes plusieurs à avoir déposé ici une traduction, il serait bon que nous ayons une discussion sur les termes à employer. Il faudra prendre en compte dans quel circonstances un terme sera employé. Pour reprendre vos 4 mots, j'aurais traduit différemment 3 d'entre eux : 
Report = signaler
Privacy = vie privée
Entries = éléments
Une collaboration serait utile il me semble...

EN
¤¤
As we are some translation providers, we should talk about our choices. We should also take in account in wich circumstances the words are used. I would have translated 3 of your 4 words differently: 
Report = signaler
Privacy = vie privée
Entries = éléments
Some collaboration would be a ggod way to handle the translation.

To the administrator of this discussion platform : Could we get all the strings that have to be translated in a single file? That would help us to discuss. Thanks in advance.

**Reply by magicvince** [reply]
Report, ma  proposition de "rapporter" est lié au substantif à utiliser ailleurs. Si signaler son substantif sera signalement. Si Rapporter son substantif sera "Rapport" (et par défaut on parle plus courament de "rapport de bug" que de "signalement" qui est généralement utilisé pour indiquer un "dérapage". Mais peut-être qu'il n'est pas nécessaire de faire le lien direct entre le verbe et le substantif et donc garder Signaler pour le verbe et Rapport pour le substantif.

Ok pour Privacy-Vie privée et Éléments pour Entries. Ces termes sont certainement plus accessibles pour différents publics.

**Comment by KrPGh** [comment]
Hello,
just finished translating missing strings in 9.1.0.44 before discovering, on submitting the file, this discussion.
Here is my file, at least for "statistical considerations" if nothing else.
I agree with Jojaba for "privacy", I would rather translate it by "vie privée"
Thanks to all the good fellows who take a bit of their precious time to contribute.
[fr.txt](https://github.com/user-attachments/files/17970631/fr.txt)

**Comment by amigainc** [comment]
Hi, here are the latest missing translations in French.
[fr.txt](https://github.com/user-attachments/files/18044971/fr.txt)

**Comment by silopolis** [comment]
Probably redundant with @amigainc
[fr.txt](https://github.com/user-attachments/files/18049364/fr.txt)
